test/ionic-app/src/components/app-home/app-home.tsx

```diff
@@ -19,13 +18,14 @@ export class AppHome {
 
         <ion-content class="ion-padding">
           <p>
-            Welcome to the PWA Toolkit. You can use this starter to build entire
```

Comment:
This one kind of surprises me, but maybe it's because I've been looking at these files for a while now 😆